homework/01LessonOne.js

```diff
@@ -17,27 +17,27 @@ var myUndefined;
 //
 // What is the difference between null and undefined?
```

Comment:
Snuck a question in here! Could you take a shot at an answer in a comment underneath?